HLTrigger/Configuration/scripts/hltIntegrationTests

```diff
@@ -332,7 +332,7 @@ if [ "$SETUP" ]; then
   #we use $MENU not $SETUP here as we force the same DB / converter as the main menu
   #this is the hltGetConfiguration behaviour and would be confusing if you had to 
   #specify converter/db on the setup menu on hltIntegration tests but not on hltGetConfiguration
-  read SETUP_Vx SETUP_DB SETUP <<< $(parse_HLT_menu "$MENU")
+  read SETUP_Vx SETUP_DB CFG <<< $(parse_HLT_menu "$MENU")
```

Comment:
```suggestion
  read SETUP_Vx SETUP_DB _ <<< $(parse_HLT_menu "$MENU")
```
Optional change: iiuc, `CFG` does not exist before, and would not be used elsewhere after this line; with `_` we could obtain the same, avoiding the edge case where `CFG` gets overwritten (if one day a `CFG` variable makes it into these scripts).

Reply:
sounds good